packages/sheriff-cli/package.json

```diff
@@ -39,7 +38,8 @@
     "publint": "^0.2.10",
     "tsconfig": "workspace:*",
     "tsup": "^8.3.0",
-    "tsx": "^4.16.2"
+    "tsx": "^4.16.2",
+    "typescript": "^5.6.3"
```

Comment:
i already did this change in my current PR, i would remove this to avoid a conflict 😊